torchgeo/losses/qr.py

```diff
@@ -14,8 +14,8 @@
 class QRLoss(Module):
     """The QR (forward) loss between class probabilities and predictions.
 
-    This loss is defined in 'Resolving label uncertainty with implicit generative
-    models' https://openreview.net/forum?id=AEa_UepnMDX.
+    This loss is defined in `'Resolving label uncertainty with implicit generative
+    models'<https://openreview.net/forum?id=AEa_UepnMDX>`_.
```

Comment:
```suggestion
    models' <https://openreview.net/forum?id=AEa_UepnMDX>`_.
```
Might be the missing space causing the build to fail?